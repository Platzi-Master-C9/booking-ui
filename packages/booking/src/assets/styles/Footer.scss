// Variables
@use "./variables.scss";

.footer {
  width: 100%;
  height: 160px;
  display: flex;
  align-items: center;
  padding-left: 64px;

  &__link {
    text-decoration: none;
    font-size: 18px;
    color: variables.$bk-black;
    margin-right: 100px;
  }
}
