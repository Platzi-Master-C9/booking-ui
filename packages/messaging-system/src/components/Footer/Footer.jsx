// External dependencies
import React from 'react';

// Internal dependencies
import classes from './Footer.module.scss';

function Footer() {
  return (
    <footer className={classes.footer}>
      <a href="/">
        Sobre nosotros
      </a>
      <a href="/">
        Prop√≥sito del proyecto
      </a>
      <a href="/">
        Redes sociales
      </a>
    </footer>
  );
}

export default Footer;
