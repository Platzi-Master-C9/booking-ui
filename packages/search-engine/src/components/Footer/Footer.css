.footer__container {
  grid-area: footer;
  width: 100%;
  height: 130px;
}

.footer__wrapper {
  max-width: 1512px;
  height: 100%;
  margin: 0 auto;
  display: flex;
  justify-content: right;
  align-items: center;
  gap: 100px;
  padding-right: 146px;
}

.link {
  font-size: 1rem;
  font-weight: 400;
  color: var(color-font-dark);
}
