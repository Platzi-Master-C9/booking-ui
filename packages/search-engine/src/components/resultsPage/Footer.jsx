import React from 'react';

export default function Footer() {
  return (
    <footer className="footer__container">
      <div className="footer__wrapper">
        <a className='link' href='./'>Sobre Nosotros</a>
        <a className='link' href='./'>Prop√≥sito del proyecto</a>
        <a className='link' href='./'>Redes Sociales</a>
      </div>
    </footer>
  )
}