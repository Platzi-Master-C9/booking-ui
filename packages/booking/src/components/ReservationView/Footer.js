import React from 'react';
// Styles
import '../../assets/styles/Footer.scss';

const Footer = () => {
  // Variables
  const tags = [
    {
      title: 'Sobre nosotros',
      link: '/home',
    },
    {
      title: 'Prop√≥sito del proyecto',
      link: '/home',
    },
    {
      title: 'Redes sociales',
      link: '/home',
    },
  ];

  return (
    <section className="footer">
      {tags
        ? tags.map((e, position) => (
            <a key={position} className="footer__link" href={e.link}>
              {e.title}
            </a>
          ))
        : null}
    </section>
  );
};

export default Footer;
