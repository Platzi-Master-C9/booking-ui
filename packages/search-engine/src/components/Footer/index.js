import React from "react";
import "./Footer.css";

const Footer = () => {
  return (
    <footer className="footer-container">
      <a href="#">Sobre nosotros</a>
      <a href="#">Prop√≥sito del proyecto</a>
      <a href="#">Redes sociales</a>
    </footer>
  );
};

export { Footer };
