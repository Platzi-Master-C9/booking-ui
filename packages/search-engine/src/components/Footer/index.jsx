import React from 'react';
import './Footer.css';

export default function Footer() {
  return (
    <footer className="footer__container">
      <div className="footer__wrapper">
        <a className="link" href="./">
          About us
        </a>
        <a className="link" href="./">
          Aim of the project
        </a>
        <a className="link" href="./">
          Social media
        </a>
      </div>
    </footer>
  );
}
