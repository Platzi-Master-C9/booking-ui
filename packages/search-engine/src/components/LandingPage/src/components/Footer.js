import React from "react";
import "../style/Footer.css";

const Footer = () => {
  return (
    <footer className="footer-container">
      <a href="#">Sobre nosotros</a>
      <a href="#">Prop√≥sito del proyecto</a>
      <a href="#">Redes sociales</a>
    </footer>
  );
};

export { Footer };
