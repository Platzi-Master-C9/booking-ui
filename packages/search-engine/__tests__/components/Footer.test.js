import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';

import Footer from '../../src/components/Footer';

describe('<Footer />', () => {
  const footer = mount(<Footer />);

  it('Render the footer component', () => {
    expect(footer.length).toEqual(1);
  });
  
  it('Render the footer links', () => {
    expect(footer.find('.link').length).toEqual(3);
    expect(footer.find('.link').at(0).text()).toEqual('About us');
    expect(footer.find('.link').at(1).text()).toEqual('Aim of the project');
    expect(footer.find('.link').at(2).text()).toEqual('Social media');
  });
});

describe('Footer Snapshot', () => {
  it('compounding the UI of the component', () => {
    const footer = create(<Footer />);
    console.log(footer.toJSON());
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
