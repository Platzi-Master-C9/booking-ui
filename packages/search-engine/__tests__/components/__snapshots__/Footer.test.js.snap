// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot compounding the UI of the component 1`] = `
<footer
  className="footer__container"
>
  <div
    className="footer__wrapper"
  >
    <a
      className="link"
      href="./"
    >
      About us
    </a>
    <a
      className="link"
      href="./"
    >
      Aim of the project
    </a>
    <a
      className="link"
      href="./"
    >
      Social media
    </a>
  </div>
</footer>
`;
